### DISCUSSION TITLE
OCR PDF with extractous

### DISCUSSION THREAD

**Discussion by rafael844** [sepinf-inc/IPED#2366] [Ideas]
Hi all, its been awhile since I dont use IPED, so i dont remember how pdf ocr works. But usually, with ocr tools, when I try to ocr a scanned pdf with embebbed text ( just a letter, or a digital signature, for example) all page is missed and ocr is not applied. With [extractous](https://github.com/yobix-ai/extractous/) you can set to ocr inline images and accomplish that. 
If IPED doesnt do that, would be nice take a look.

**Comment by lfcnassif** [comment]
IPED does, not by default. Just set conf/ParsingTaskConfig.txt->processImagesInPdfs = true

**Comment by rafael844** [comment]
I didnt know that, thank you @lfcnassif !
